README.md

```diff
@@ -1,105 +1,80 @@
-
-## How to test the software
-
-If the software includes automated tests, detail how to run those tests.
-
-## Known issues
-
-Document any known significant shortcomings with the software.
-
-## Getting help
-
-Instruct users how to get help with this software; this might include links to an issue tracker, wiki, mailing list, etc.
-
-**Example**
-
-If you have questions, concerns, bug reports, etc, please file an issue in this repository's Issue Tracker.
-
-## Getting involved
-
-This section should detail why people should get involved and describe key areas you are
-currently focusing on; e.g., trying to get feedback on features, fixing certain bugs, building
-important pieces, etc.
-
-General instructions on _how_ to contribute should be stated with a link to [CONTRIBUTING](CONTRIBUTING.md).
-
-
-----
-
-## Open source licensing info
-1. [TERMS](TERMS.md)
-2. [LICENSE](LICENSE)
-
-
-----
-
-## Credits and references
-
-1. Projects that inspired you
-2. Related projects
-3. Books, papers, talks, or other sources that have meaningful impact or influence on this project
+# Basic Model Interface (BMI) for potential evapotranspiration (PET) functions:
+* Aerodynamic method
+* Combination method
+* Energy balance method
+* Penman Monteith method
+* Priestley Taylor method
+
+# The configuration file (./configs/pet_config*.txt)
+Many options are available when running this PET module. One option is passed into the executable, and that is the speficic PET method option, the rest are passed in through the configuration file. These include forcing data (type and location), vegetation characteristics, site latitude/longitude/elevation, turbidity roughness and options about the forcings available vs calculated and assumed. Each instance of a PET model should have its own unique configuration file. 
+
+# Compiling this code
+The BMI functionality was developed as a standalone module in C. To compile this code the developer used these steps:
+1. `module load gnu/10.1.0`
+2. `gcc -lm ./src/main_read_forcing.c ./src/pet.c ./src/bmi_pet.c -o run_bmi`
+This should generate an executable called **run_bmi**. To run this executable you must pass the path to the corresponding configuration file, which includes the PET method you would like to run. Unit tests for those methods, and corresponding are provided, and can be run using:
+1. Energy balance method: `./run_bmi pet_config_unit_test1.txt`
+2. Aerodynamic method: `./run_bmi pet_config_unit_test2.txt`
+3. Combination method: `./run_bmi pet_config_unit_test3.txt`
+4. Priestley Taylor method: `./run_bmi pet_config_unit_test4.txt`
+5. Penman Monteith method: `./run_bmi pet_config_unit_test5.txt`
+Included in this repository is an environment file (env_cheyenne.sh), and a "make and run" file (make_and_run.sh), which will compile the code and run the five PET methods. If you are on the Cheyenne computer, or if you can modify these files to your machine, you can simply follow these two steps to run this code:
```

Comment:
add newline space between ln 20 & 21 to format [list](https://github.com/NOAA-OWP/evapotranspiration/blob/4f04e161893a0ec24b6b818e9c08399931110270/README.md#compiling-this-code) correctly.